mkdocs.yml

```diff
@@ -105,6 +105,7 @@ theme:
         icon: material/weather-night
         name: Switch to light mode
   features:
+    - content.tabs.link
```

Comment:
This makes it so that if a user picks an option on a page (like "Model" or "Tag" in this example) then it updates all of the codeblocks to match that selection.